Hit.h

```diff
@@ -100,6 +100,10 @@ inline float getEta(float r, float z) {
   return -1.0f * std::log( std::tan(getTheta(r,z)/2.) );
 }
 
+inline float getEta(float theta){
+  return -1.0f * std::log( std::tan(theta/2.) );
```

Comment:
should be 2.f, same for line 100 (that ones my fault, I missed it when doing the scan for doubles)